x-pack/test/functional/services/uptime/navigation.ts

```diff
@@ -17,7 +17,7 @@ export function UptimeNavigationProvider({ getService, getPageObjects }: FtrProv
       if (await testSubjects.exists('uptimeSettingsToOverviewLink', { timeout: 0 })) {
         await testSubjects.click('uptimeSettingsToOverviewLink');
         await testSubjects.existOrFail('uptimeOverviewPage', { timeout: 2000 });
-      } else if (!(await testSubjects.exists('uptimeOverviewPage', { timeout: 0 }))) {
```

Comment:
I think this isn't related to the alert test flakiness, it's related to the EuiLink not being able to click.

Reply:
I am pretty sure when I was testing this, skipping the contents of this block were causing the test to fail. I can push a commit that reverts this change to see if it keeps succeeding.

Reply:
@shahzad31 I am going to need to wait a bit to test this because Jenkins maintenance is happening right now.